### DISCUSSION TITLE
Extracting labels

### DISCUSSION THREAD

**Discussion by ShriNear** [opengeos/segment-geospatial#189] [Q&A] [answered]
Hi,
Is there any way that  we can use SAM for object detection and segmentation like yolo and other models, were they take image as input and give us output as masks or bounding box of objects with their class labels?

**Comment by giswqs** [comment, marked as answer]
Yes, you can use bounding boxes from other models as input to SAM. See https://samgeo.gishub.org/examples/box_prompts/

**Reply by ShriNear** [reply]
Thank you! I will look into it

**Reply by ShriNear** [reply]
I think in this method I have to draw bounding boxes for every image but I want it like, Just give input image and it gives output as bounding boxes or mask and corresponding labels.
